test/Land/Model/test_radiative_energy_flux_functions.jl

```diff
@@ -0,0 +1,32 @@
+# Test functions used in runoff modeling.
+using MPI
+using OrderedCollections
+using StaticArrays
+using Test
+
+using ClimateMachine.Land.RadiativeEnergyFlux
+using ClimateMachine.VariableTemplates
+
+
+@testset "Radiative energy flux testing" begin
+    F = Float32
+    user_nswf = t -> F(2 * t)
+    user_swf = t -> F(2 * t)
+    α = F(0.3)
+    prescribed_swf_and_a = PrescribedSwFluxAndAlbedo(F; α = α, swf = user_swf)
```

Comment:
same question of if alpha is a float or a function of time

Reply:
keeping (t) for now? see above comment

Reply:
sounds good!